index.js

```diff
@@ -34,6 +39,22 @@ var cli = new Liftoff({
   completions: completion,
   extensions: interpret.jsVariants,
   v8flags: v8flags,
+  configFiles: {
+    '.gulprc': {
+      home: {
+        path: '~',
+        extensions: interpret.extensions,
```

Comment:
we only want to support the `jsVariants` so we should allow this to be defaulted by the top-level `extensions`.  No need to specify again.